neural_compressor/experimental/pytorch_pruner/prune_utils.py

```diff
@@ -189,7 +221,7 @@ def process_config(config):
     return process_and_check_config(val)
 
 
-def parse_to_prune(model, config):
+def parse_to_prune(config, model):
```

Comment:
why the parameter's order is changed? any special reason?

Reply:
In order to unify the order of the two formal parameters in the internal functions, the change does not affect the pruner API.